### ISSUE TITLE
trying to FORCE delete a pod fails with `Error deleting converting (url.Values) to (v1.DeleteOptions): unknown conversion`

### ISSUE THREAD

**Issue by wc-matteo** [rancher/rancher#32847]
while delete without forcing works as expected

```
K8s v1.21.1 (Scaleway Kubernetes Kapsule)
Rancher v2.5.8
```

gz#17810

**Comment by anjia0532** [comment]
same
```
K3Sv1.20.6+k3s1
rancher v2.5.8
```

**Comment by timmy59100** [comment]
Same. works with kubectl but not in UI.

**Comment by tuxpeople** [comment]
Same here with Rancher 2.5.8 and 2.5.7 on RKE1 Downstream Clusters. Works via `kubectl`.

**Comment by antoinetran** [comment]
Reproduced with Rancher 2.5.7. Thanks for the workaround, we will apply that.

**Comment by Napsty** [comment]
Can confirm. Rancher 2.5.8, Single Install (using k3s in the background).

![image](https://user-images.githubusercontent.com/3702203/123422069-f3c15100-d5bd-11eb-9eea-0fcd3cd4cd04.png)

**Comment by kaneleven** [comment]
Your kubelet must have issue, try restart it

**Comment by tuxpeople** [comment]
> Your kubelet must have issue, try restart it

That is a thoughtful, very profound answer.

No, seriously, I very much doubt it. Because via the CLI it works without any problems. The problem exists only in the UI.

**Comment by thbiela** [comment]
Issue still persists with Rancher 2.5.9 and k8s v1.20.6

**Comment by jonathon2nd** [comment]
I am hoping that this can be addressed with 2.6 upgrade :crossed_fingers: 

Still seeing in:

Rancher | v2.5.9
User Interface | v2.5.9
Helm | v2.16.8-rancher1
Machine | v0.15.0-rancher55

**Comment by jonathon2nd** [comment]
Looks like `force delete` is no longer a _thing_ in 2.6.
![Screenshot_20210915_112529](https://user-images.githubusercontent.com/52681917/133480811-b078f1b0-beda-414d-8def-c3c525e367b2.png)

**Comment by anjia0532** [comment]
rancher wechat group 

![image](https://user-images.githubusercontent.com/15098916/133543770-e372927e-bd58-48ba-b657-092768db4da3.png)

**Comment by tuxpeople** [comment]
> Looks like `force delete` is no longer a _thing_ in 2.6.
> ![Screenshot_20210915_112529](https://user-images.githubusercontent.com/52681917/133480811-b078f1b0-beda-414d-8def-c3c525e367b2.png)

lol. They just removed the "force" option to fix that?

**Comment by niusmallnan** [comment]
I have filed a PR to fix this issue.
A fix for 2.5 is required, 2.6 is optional(as we don’t use Norman API to manage the k8s resources).
Hope that we can merge it to the next v2.5.x milestone.

**Comment by brudnak** [comment]
# :white_check_mark: PASSED

## Reproduction Environment

| Component               | Version / Type   |
| ----------------------- | ---------------- |
| Rancher version         | 2.5.10           |
| Installation option     | docker           |
| Cert Details            | self signed      |
| Helm version            | v2.16.8-rancher1 |
| Downstream cluster type | rke1 linode      |
| Downstream K8s version  | v1.20.11         |
| Logged in user role     | administrator    |
| Browser type            | google chrome    |
| Browser version         | 102.0.5005.115   |


## Reproduction steps

1. Create a downstream cluster
2. Create a deployment
3. Force delete a pod from the deployment
4. Receive the following error:

```log
Error deleting
converting (url.Values) to (v1.DeleteOptions): unknown conversion
```

## RESULTS

#### :white_check_mark: Expected
Force delete to remove the pod

#### :x: Actual
Pod was not deleted and an error message was displayed

<hr>
  
## Validation Environment

| Component                 | Version / Type                                                                     |
| ------------------------- | ---------------------------------------------------------------------------------- |
| Rancher version           | v2.6-a8ae635611f1fb431f81f48ec19997fbc17c746f-head                                 |
| Rancher commit link       | https://github.com/rancher/rancher/commit/a8ae635611f1fb431f81f48ec19997fbc17c746f |
| Installation option       | helm                                                                               |
| RKE binary version used   | 1.3.11                                                                             |
| If Helm Chart k8s cluster | rke v1.23.6                                                                        |
| Cert Details              | let's encrypt                                                                      |
| Docker version            | 20.10.7, build f0df350                                                             |
| Helm version              | v2.16.8-rancher1                                                                   |
| Downstream cluster type   | rke1 linode                                                                        |
| Downstream K8s version    | v1.23.7                                                                            |
| Cert manager version      | v1.7.1                                                                             |
| Auth Provider             | github                                                                             |
| Logged in user role       | administrator                                                                      |
| Browser type              | google chrome                                                                      |
| Browser version           | 102.0.5005.115                                                                     |

## Validation steps

1. Create a downstream cluster
2. Create a deployment
3. Force delete a pod from the deployment from `/g`
4. Deletion is successful with no errors
  
## Additional Info

## RESULTS

#### :white_check_mark: Expected
Deletion to work without errors

#### :white_check_mark: Actual
Deletion worked without errors
